onchain/src/social_account.cairo

```diff
@@ -0,0 +1,20 @@
+#[starknet::contract]
+pub mod SocialAccount {
+    #[storage]
+    struct Storage {
+        public_key: felt252,
```

Comment:
`public_key` should be `u256`. See: https://github.com/keep-starknet-strange/joyboy/blob/4c572d060c946efb74b23f661a2576fe1cc1421f/onchain/src/bip340.cairo#L31C14-L31C18.

Reply:
Add a view method: `get_public_key`.